scripts/linters/custom_eslint_checks/rules/protractor-practices.js

```diff
@@ -159,6 +162,13 @@ module.exports = {
           messageId: 'disallowThen'
         });
       },
+      'CallExpression[callee.property.name=\'forEach\']': function(node) {
```

Comment:
```suggestion
      'CallExpression[callee.property.name=forEach]': function(node) {
```
Will this work? If yes, let's keep it this way! :)